price-estimator/docker/deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+image_name=$1
```

Comment:
nit: In Docker lingo, I believe this is called a "tag". I find calling the variable "image_name" a bit confusing (to me the image name is "dex-price-esimator-rust"):

```suggestion
image_tag=$1
```

Reply:
Will change it to that. Reason for this name is that it's the same as in the driver's deploy script.